### PR TITLE
feat(Stepper): stepper

### PR DESCRIPTION
**Pull request by ItamarGronich**
### Description
Add stepper.

PR Resources | Links
------ | ------
🎫 Ticket | [ACTIVE-690](https://fundbox.atlassian.net/browse/ACTIVE-690)

### CODE REVIEW

File: src/components/FbxStepper/FbxStepper.vue [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+<script>
+export default {
+  name: 'FbxStepper',
+  props: {
+    steps: {
+      type: Array,
+      required: true,
+    },
+    activeStep: {
+      type: Number,
```
Comment (forforeach) [inline src/components/FbxStepper/FbxStepper.vue]: Put 0 as default

Comment (ItamarGronich) [reply src/components/FbxStepper/FbxStepper.vue]: Are you sure we want the first step to be active by default?

Comment (ItamarGronich) [reply src/components/FbxStepper/FbxStepper.vue]: I say no because we should allow no active steps i think.

Comment (forforeach) [reply src/components/FbxStepper/FbxStepper.vue]: What is this strange state when nothing is active? Like you landed in to some page that has a stepper but you didn't start yet? What is the reason of showing the stepper if this isn't a first step at least?

**Pull request merged**